rcl/include/rcl/wait.h

```diff
@@ -35,10 +35,12 @@ extern "C"
 #include "rcl/types.h"
 #include "rcl/visibility_control.h"
 
-struct rcl_wait_set_impl_t;
+struct rcl_wait_set_impl_s;
+
+typedef struct rcl_wait_set_impl_s rcl_wait_set_impl_t;
```

Comment:
Same question about removing the typedef.

Reply:
Same as above.